operator/pkg/bgpv2/cell.go

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package bgpv2
+
+import (
+	"github.com/spf13/pflag"
+
+	"github.com/cilium/cilium/pkg/hive/cell"
+)
+
+const (
+	// BGPv2Enabled is the name of the flag that enables BGPv2 APIs in Cilium.
+	BGPv2Enabled = "enable-bgp-v2-api"
+)
+
+var Cell = cell.Module(
+	"bgp-cp-operator",
+	"BGP Control Plane Operator",
+	cell.Config(Config{}),
```

Comment:
clarification: I assume default is intended to be false here (no change necessary, just checking)?

Reply:
Yes, by default bgpv2 is disabled.